script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -195,7 +195,7 @@ The girls stare at them coldly with a gaze that says 'boys are pathetic'. But th
 -- Page 138, Offset 2963.
 -- 「トビウオみたいな反応、ありがとう。
 // トビウオみたいな反応、ありがとう。Dobiuo is a flying fish, it's also used in ドビウオ女 as a contrast to マグロ女 ("tuna women", starfish, dead lay), and means that she's eager to please (in bed, but apparently also in general). I don't have, and there may not be, anything to keep the reference (and while マグロ女 is pretty famous, ドビウオ女 seems to be really obscure honestly). Need to add italics to "please" when possible
-"I love the eager-to-please attitude, thanks guys!
+"I love the eager-to-%{i}please%{/i} attitude, thanks guys!
```

Comment:
because of the italics, I'd say drop the hyphens and swap "attitude" for "reaction"